assets/src/js/admin/stripe-admin.js

```diff
@@ -27,13 +27,145 @@ window.addEventListener( 'DOMContentLoaded', function() {
 	const becsIconStyleElement = document.querySelector( '.stripe-becs-icon-style' );
 	const hideBecsMandateElements = Array.from( document.querySelectorAll( ' input[name="stripe_becs_mandate_acceptance_option"]' ) );
 	const mandateBecsElement = document.querySelector( '.stripe-becs-mandate-acceptance-text' );
+	const manualFields = Array.from( document.querySelectorAll( '.give-stripe-account-type-manual' ) );
+	const connectField = document.querySelector( '.give-stripe-account-type-connect' );
+	const connectionTypes = Array.from( document.querySelectorAll( 'input[name="stripe_connection_type"]' ) );
+	const selectedConnectionType = document.querySelector( 'input[name="stripe_connection_type"]:checked' );
+	const disconnectBtns = Array.from( document.querySelectorAll( '.give-stripe-disconnect-account-btn' ) );
+	const setStripeDefaults = Array.from( document.querySelectorAll( '.give-stripe-account-set-default' ) );
+	const perFormOptions = Array.from( document.querySelectorAll( 'input[name="give_stripe_per_form_accounts"]' ) );
+	const perFormAccount = document.querySelector( '.give-stripe-per-form-default-account' );
+	const perAccountEdits = Array.from( document.querySelectorAll( '.give-stripe-account-edit-name' ) );
+	const perAccountUpdates = Array.from( document.querySelectorAll( '.give-stripe-account-update-name' ) );
+	const accountManagerError = document.getElementById( 'give-stripe-account-manager-errors' );
 
 	giveStripeJsonFormattedTextarea( stripeStylesBase );
 	giveStripeJsonFormattedTextarea( stripeStylesEmpty );
 	giveStripeJsonFormattedTextarea( stripeStylesInvalid );
 	giveStripeJsonFormattedTextarea( stripeStylesComplete );
 	giveStripeJsonFormattedTextarea( stripeCustomFonts );
 
+	if ( null !== perAccountEdits ) {
```

Comment:
When doing direct DOM manipulation it is especially important to leave comments to document what's going on, as it is by no means obvious what's happening. Can you please go through this and leave a comment here and there to indicate what each hunk of code is doing?

Reply:
Added detailed comments